cumulus/tasks/pdr-status-check/index.js

```diff
@@ -130,6 +131,21 @@ function buildOutput(event, groupedExecutions) {
   return output;
 }
 
+/**
+ * check the status of a step funciton execution
+ *
+ * @param {string} executionArn - step function execution arn
+ * @returns {Promise.<Object>} - an object describing the status of the exection
+ */
+function describeExecutionStatus(executionArn) {
+  return aws.sfn().describeExecution({ executionArn }).promise()
+    .catch((e) => {
+      if (e.code === 'ExecutionDoesNotExist') {
+        return { executionArn: executionArn, status: 'RUNNING' };
```

Comment:
Can you explain this to me? If we get an error `ExecutionDoesNotExist` why do we return a status of `RUNNING`?

Reply:
The granule stats in the pdr have the following lists: running, completed, failed.  Initially, the PDR queues these granules and puts the executions to running list.  So, the executions which haven't been executed are still belong to running list.